CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-## 2.8.0-rc.0 / 2019-03-06
+## 2.8.0 / 2019-03-11
+
+This release uses WAL for remote_write API. This currently causes a little higher usage of memory, which will be improved upon in the future releases.
```

Comment:
Nit: I think it would be nice to specify Write-ahead log here. Unexpanded acronyms make me very sad.

Reply:
No problem with that. Doing.